File: Java/Lista_01(Estrutura_Sequencial)/Pagamento.java
import java.util.Locale;
import java.util.Scanner;

public class Pagamento {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Nome: ");
		String nome = s.nextLine();
		
		System.out.print("Valor por hora: ");
		double valorPorHora = s.nextDouble();
		
		System.out.print("Horas trabalhadas: ");
		int horasTrabalhadas = s.nextInt();
		
		double pagamento = valorPorHora * horasTrabalhadas;
		
		System.out.printf("O pagamento para %s deve ser %.2f", nome, pagamento);
		
		s.close();
		
	}

}

File: Python/Lista_01(Estrutura_Sequencial)/Pagamento.py
nome = input("Nome: ")

valorPorHora = float(input("Valor por hora: "))

horasTrabalhadas = int(input("Horas trabalhadas: "))

pagamento = valorPorHora * horasTrabalhadas

print(f"O pagamento para {nome} deve ser {pagamento:.2f}")